data/hai/unfettered hunger.txt

```diff
@@ -0,0 +1,395 @@
+			`	She hesitates but then the baby starts to cry. She looks at you with tears in her eyes and nods. You send some crew with her to help her pack her things and bring them to the ship.`
+				accept
+			
+			label business
+			`	The Unfettered mother never notices you as you pass by. You see the family later in a long queue for food rations.`
+				decline
+
+			label condolences
+			`	You tell her you are sorry for her loss. You see anger flare across her features as you turn to leave and if she wasn't holding an infant she might well have attacked you. "What good are your words monkey-man?" she shouts. "Will they feed my children? Did you come all this way to make fun of us? We have to choose between dying for a new world or starving on this one. Go back through the wormhole and give your words to your own kind!"`
+			`	A group of Unfettered roughly screen you off from the distraught mother and tells you to leave. She continues screaming at you and crying as you walk away.`
+				decline
+
+	on complete
+		payment 300000
+		conversation
+			`You are surprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill that your crewmember says is a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for brining our family back together" and gives you your payment of <payment>.`
+
+
+
+mission "Unfettered: Hunger Stories 2"
+	minor
+	name `Transport a Hai warrior to <destination>`
+	description `Transport an Unfettered warrior to <destination>.`
+	source "Darkcloak"
+	destination "Makerplace"
+	passengers 1
+	to offer
+		has "unfettered: hunger"
+	on offer
+		conversation
+			`"Human!" The cry catches your attention not only because of the volume of the shout, but primarily because it's the most respectful term anyone has used for you in Unfettered space. You turn and see a young Hai warrior striding toward you. At least you think he's young. He's certainly thinner than most of the Hai you see south of Wah-Ki. You notice that his boots seem made more of mending tape than material. He's not wearing the ornately decorated knives that many of the True Hai carry, but he does have sheaths for them. Instead, he carries a long staff that looks to be made of one of the light but rigid Hai alloys.You can see that scars from many fights disturb the pattern of his fur.`
+			`	"Human," he says again when he reaches you, "you should not promenade yourself in our star market without worthy guidance. This warrior can tell you what trader tongues are saying because I know your backward speech. Fast fingers not grab human things while I watch," he adds, emphasing the point by banging the staff on the floor and making it ring. You notice the flow of traffic begins to part around you as other traders and travellers give him room.`
```

Comment:
```suggestion
			`	"Human," he says again when he reaches you, "you should not promenade yourself in our star market without worthy guidance. This warrior can tell you what trader tongues are saying because I know your backward speech. Fast fingers not grab human things while I watch," he adds, emphasizing the point by banging the staff on the floor and making it ring. You notice the flow of traffic begins to part around you as other traders and travellers give him room.`
```